src/Service.Tests/SqlTests/RestApiTests/Insert/MsSqlInsertApiTests.cs

```diff
@@ -297,7 +297,7 @@ public async Task InsertOneInViewBadRequestTest()
                 operationType: EntityActionOperation.Execute,
                 requestBody: requestBody,
                 expectedStatusCode: HttpStatusCode.Created,
-                expectedLocationHeader: _integrationProcedureInsertOneAndDisplay_EntityName,
+                expectedLocationHeader: string.Empty,
```

Comment:
We already have some location header checks. Arent those sufficient? I remember our base tests had been modified to simulate real client server request responses as well.. Cant we send the followup response in these integration tests? 

Do we really need to add additional tests to the ConfigurationTests?

Reply:
The current checks we have validates only the PK part of the Location header. The entire field was not validated.
a) Adding tests in Configuration Tests lets us modify the config to add a custom base route which gets used only in that test. We need this for the scenario where base-route is configured.
b) In the existing integration tests, we have a common validation method for all Insert API tests. The current check validates only the PK part of the location header and not the entire field. Performing a GET follow-up request there would mean that the GET action should be enabled for the entities (and we need to ensure that entities that are added in the future would also have GET APIs enabled). Or introduce another flag in the tests to essentially indicate if a follow up GET request needs to be performed.

Considering these, I've added new tests in ConfigurationTests